symfony/notifier/5.0/config/packages/notifier.yaml

```diff
@@ -0,0 +1,16 @@
+framework:
+    notifier:
+        #chatter_transports:
+        #    slack: '%env(SLACK_DSN)%'
+        #    telegram: '%env(TELEGRAM_DSN)%'
+        #texter_transports:
+        #    twilio: '%env(TWILIO_DSN)%'
+        #    nexmo: '%env(NEXMO_DSN)%'
+        channel_policy:
+            # use chat/slack, chat/telegram, sms/twilio or sms/nexmo
```

Comment:
I don’t know the component yet, but it’s not clear where these available options come from and why email is also an option. The chatter and texter above don’t match the chat/sms here. This might not be the right place for the feedback - just passing my novice questions :)

Reply:
We have severals channels: sms, chatter, email. Each channel might have specific transports, so you have chat/slack and chat/telegram.

Reply:
That'd mean we can use: `urgent: ['email', 'sms/twilio']` where `sms` is the channel and `twilio` the transport, right?